src/index.html

```diff
@@ -1,13 +1,273 @@
+                    <img
+                      src="images/details-4.png"
+                      alt="detail"
+                      class="category__photo"
+                    >
+                  </a>
+                </div>
+                <h3 class="category__title">Range & Integrated lights</h3>
+                <p class="category__description">
+                  The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+                </p>
+              </article>
+
+              <article class="category">
+                <div class="category__photos">
+                  <a href="#" class="category__link category__link--square">
+                    <img
+                      src="images/details-5.png"
+                      alt="detail"
+                      class="category__photo"
+                    >
+                  </a>
+
+                  <a href="#" class="category__link category__link--wide">
+                    <img
+                      src="images/details-6.png"
+                      alt="detail"
+                      class="category__photo"
+                    >
+                  </a>
+                </div>
+                <h3 class="category__title">Hydraulic disc brakes & Lightweight</h3>
+                <p class="category__description">
+                  Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+                </p>
+              </article>
+            </div>
+            <button class="details__button">Explore</button>
+          </section>
+
+          <section class="contacts" id="contacts">
+            <h2 class="section-title contacts__title">Contact us</h2>
+
+            <div class="contacts__wrapper">
+              <div class="form">
+                <form
+                  class="form__block"
+                  type="text"
+                  name="message"
+                  required
+                  placeholder="Message"
+                  minlength="1"
+                  rows="4"
+                >
+                  <input
+                    class="form__input"
+                    type="text"
+                    id="name"
+                    name="name"
+                    placeholder="Name"
+                  >
+                  <input
+                    class="form__input"
+                    type="email"
+                    id="email"
+                    name="email"
+                    placeholder="Email"
+                  >
+                  <textarea
+                    class="form__text-area"
+                    type="text"
+                    id="message"
+                    name="message"
+                    placeholder="Message"
+                  >
+                  </textarea>
```

Comment:
```suggestion
                  <textarea
                    class="form__text-area"
                    type="text"
                    id="message"
                    name="message"
                    placeholder="Message"
                  ></textarea>
```

Fix format to remove extra spaces in textarea field